src/main/resources/func.json

```diff
@@ -1,30 +1,30 @@
 {
   "tools": {
     "func": {
-      "version": "0.26.0",
+      "version": "1.7.0",
```

Comment:
@lstocchi For func cli, 1.7.1 is also out. We can migrate to that version.

Reply:
I'm going to create a new issue to bump it.